Ryujinx.Input.SDL2/SDL2Gamepad.cs

```
@@ -151,7 +151,18 @@ public void Rumble(float lowFrequency, float highFrequency, uint durationMs)
                 ushort lowFrequencyRaw = (ushort)(lowFrequency * ushort.MaxValue);
                 ushort highFrequencyRaw = (ushort)(highFrequency * ushort.MaxValue);
 
-                SDL_GameControllerRumble(_gamepadHandle, lowFrequencyRaw, highFrequencyRaw, durationMs);
+                if (durationMs == uint.MaxValue)
+                {
+                    SDL_GameControllerRumble(_gamepadHandle, lowFrequencyRaw, highFrequencyRaw, SDL_HAPTIC_INFINITY);
+                }
+                else if (durationMs > SDL_HAPTIC_INFINITY)
+                {
+                    throw new NotSupportedException($"Unsupported rumble duration {durationMs}");
```

I’m not sure we should throw an exception here, maybe a logging error is enough, since it will not stop the execution for a vibration error.